lib/message_handler.rb

```diff
@@ -42,6 +42,17 @@ def handle
         )
 
         submit_collection_updater.update_scsb_items
+
+        mailer = ErrorMailer.new(
+          error_hashes: [mapper.errors, submit_collection_updater.errors],
```

Comment:
probably should include the error from scsbxml_fetcher too. Also, we could check in case no errors then we don't have to execute `mailer.send_error_email`.

Reply:
Exactly!

Let's coordinate. I can:

* accept your PR (https://github.com/NYPL-discovery/scsb_item_updater/pull/16).
* rebase `master` into this branch.
* Add it to this collection.

Reply:
sounds a good plan!